mods/creative/inventory.lua

```diff
@@ -1,5 +1,10 @@
 local player_inventory = {}
 
+-- Creates a blank detached inventory to be filled with contents later
```

Comment:
:+1: for doc-comments, but is this LuaDoc/LDoc, Doxygen, or what?

For LuaDoc/LDoc it should be three dashes (`---`). Not sure about Doxygen.